conda/meta.yaml

```diff
@@ -43,32 +60,27 @@ test:
     - pandas
     - scipy>=1.7.0
     - tbb-devel
-    - pandoc
-    - sphinx
-    - sphinx-book-theme
-    - sphinx-copybutton
-    - nbsphinx
   files:
     - cmake-package-test/
   source_files:
     - tests/
-    - docs/
-    - .git/  # Docs build uses git to determine version and release month
-  script_env:
-    - DOCS_BUILD_DIR
   commands:
     - python -m pytest -v tests
     - python cmake-package-test/build.py
-    - python docs/build_and_test.py [linux64]
 
 build:
   # Build number is number of Git commits since last tag, if that fails use 0
   number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
   script_env:
-    - OSX_VERSION [osx]
-    - CMAKE_INTERPROCEDURAL_OPTIMIZATION=ON
+   - DOCS_HTML_DIR # [linux]
```

Comment:
What is the value of this variable?

Reply:
It is set in the package build workflow file, to  have a mechanism of finding where the docs output is placed.